themes/10up-theme/tailwind.config.js

```diff
@@ -10,32 +34,60 @@ module.exports = {
 	 */
 	content: [
 		/* Ensure changes to all PHP, JS, and JSON files rebuild your CSS */
-		'./assets/js/**/*.js',
-		'./includes/**/*.{php,js,json}',
-		'./partials/**/*.php',
-		'./templates/**/*.php',
+		'404.php',
+		'archive.php',
+		'attachment.php',
+		'author.php',
+		'category.php',
+		'date.php',
+		'footer.php',
+		'front-page.php',
+		'functions.php',
+		'header.php',
+		'home.php',
+		'index.php',
+		'page.php',
+		'search.php',
+		'searchform.php',
+		'single.php',
+		'singular.php',
+		'tag.php',
+		'taxonomy.php',
+
+		// Directories
+		'assets/**/*.js',
+		'includes/**/*.{js, php}',
+		'partials/**/*.php',
+		'templates/**/*.php',
 	],
 	safelist: [
-		/* Prevent editor-specific styles from being purged */
-		'editor-post-title__block',
-		'editor-post-title__input',
-		'entry-content',
-		'entry-title',
-		'block-editor-block-list__layout',
+		{
+			pattern: /(bg|text)-(primary|secondary|tertiary)/,
+		},
 	],
 	theme: {
 		/* Override the default theme */
-
+		container: {
+			center: true,
+			padding: {
+				default: '1.25rem',
+				md: '2rem',
+				lg: '2.5rem',
+			},
+		},
 		/* Define brand colors using --wp custom properties from the theme.json */
 		colors: {
 			transparent: 'transparent',
 			current: 'currentColor',
-			white: 'var(--wp--preset--color--white)',
-			black: 'var(--wp--preset--color--black)',
+			...getThemeColors(),
+			/* Include additional palette colors as appropriate to fill in gaps for brand colors */
+			gray: colors.gray,
 		},
 		extend: {
 			/* Extend the default theme */
-
+			screens: {
+				'wp-md': '600px' /* WP Specific Breakpoint used in Gutenberg */,
```

Comment:
Great add - would it be worth while aligning or adding specified wp media query values to this config in the long run?

Reply:
I think we should absolutely include all core breakpoints if we can. This way we can do things like: 

`admin-md:hidden` or stuff like that. Our "toolbox" of styles should include anything we'd reasonably want to do on the frontend and ideally flow well with any block styles from core, so that seems like a natural next step.